zkevm-circuits/src/evm_circuit/util/common_gadget/tx_eip1559.rs

```diff
@@ -144,6 +180,29 @@ impl<F: Field> TxEip1559Gadget<F> {
             offset,
             Some(tx.max_priority_fee_per_gas.to_le_bytes()),
         )?;
+
+        let diff_gas_base_fee = tx.max_fee_per_gas - base_fee;
+        let priority_fee_per_gas = base_fee + tx.max_priority_fee_per_gas.min(diff_gas_base_fee);
```

Comment:
priority_fee_per_gas => effective_gas_price

Reply:
updated in [e1a0d3b](https://github.com/scroll-tech/zkevm-circuits/pull/1152/commits/e1a0d3bcf142705e30eb6dee1b893d4ff0415850)